.storybook/preview.js

```diff
@@ -1,26 +1,26 @@
-import IronFishUIProvider from 'components/IronFishUIProvider';
-import ColorModeSwitcher from 'components/ColorModeSwitcher';
-import { Flex } from "@chakra-ui/react"
+import IronFishUIProvider from 'components/IronFishUIProvider'
+import ColorModeSwitcher from 'components/ColorModeSwitcher'
+import { Box, CSSReset, Flex } from '@chakra-ui/react'
```

Comment:
is `CSSReset` being used? I'd like to add some pre-commit hooks like `nps lint`, would you and @azhurauski-sc be ok with that?  (I think it'll decrease a bunch of our problems, but I'd be ok with moving it to a github workflow instead -- I personally prefer pre-commit hooks but I don't feel strongly)

Reply:
Good idea about pre-commit hook with `nps lint`.

Reply:
Also like that idea as I used to rely on IDE lint, but sometimes it might miss something.